bindings/bindingtester/tests/directory.py

```diff
@@ -90,6 +90,8 @@ def generate(self, args, thread_number):
         directory += directory_reads
         directory += directory_db_mutations
         directory += directory_db_reads
+
+        if not args.no_directory_snapshot_ops:
```

Comment:
Are the contents of this if-block not indented?

Reply:
Oops, looks like I had a tooling issue. Fixed.